mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulSettingsScreen.kt

```diff
@@ -3,22 +3,19 @@ package ch.epfl.sdp.mobile.state
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
-import ch.epfl.sdp.mobile.application.Profile.Color
 import ch.epfl.sdp.mobile.application.authentication.AuthenticatedUser
 import ch.epfl.sdp.mobile.ui.setting.SettingScreenState
 import ch.epfl.sdp.mobile.ui.setting.SettingsScreen
 import ch.epfl.sdp.mobile.ui.social.ChessMatch
+import ch.epfl.sdp.mobile.ui.social.Person
 
-class AuthenticatedUserProfileScreenState(private val user: AuthenticatedUser) :
-    SettingScreenState {
+class AuthenticatedUserProfileScreenState(
+    private val user: AuthenticatedUser,
+) : SettingScreenState, Person by ProfileAdapter(user) {
```

Comment:
[Question] Why make the AuthenticatedUserProfileScreenState a Person?

Reply:
It's already a `Person`, since `SettingScreenState : Person`.